packages/cli/test/dev/fixtures/test-rewrites/vercel.json

```diff
@@ -9,5 +9,15 @@
       }
     }
   ],
-  "rewrites": [{ "source": "/hello", "destination": "index.html" }]
+  "rewrites": [
+    {
+      "source": "/hello",
+      "destination": "index.html"
+    },
+    {
+      "source": "/status-rewrite-425",
+      "destination": "index.html",
+      "statusCode": 425
```

Comment:
I'm not sure I understand the use case for this feature. Why would you need to change the status code for a rewrite to a static file? And if its not static, seems like you could return the correct status code from the function instead.

Reply:
Originally I had an example with `404` which is a more practical use case for that.
I decided to use the `425` status to make sure the tests don't pass if mistakenly it hits a non-existing route which would produce a `404` automatically.